### PR TITLE
Print log server information on plugin run

### PR DESCRIPTION
**Pull request by mgdelacroix**
#### Summary
This change prints the same information that the standalone instance does on plugin activation

### CODE REVIEW

**Review by sbishel — APPROVED** [review]

File: mattermost-plugin/server/plugin.go [hunk 1]
```diff
@@ -52,6 +53,8 @@ func (p *Plugin) OnActivate() error {
 		return fmt.Errorf("cannot activate plugin: %w", err)
 	}
 
+	model.LogServerInfo(logger)
```
Comment (wiggin77) [inline mattermost-plugin/server/plugin.go:56]: @mgdelacroix can you add this to product mode as well? (https://github.com/mattermost/focalboard/blob/feb49eaf197afed8c082735837ebf95e04210621/mattermost-plugin/product/boards_product.go#L200)

Comment (mgdelacroix) [reply mattermost-plugin/server/plugin.go:56]: Done on [b087f27](https://github.com/mattermost/focalboard/pull/3711/commits/b087f27ddf8277471c8405f296b5a42d8c732969), please let me know if it's the right place to add it

Comment (mgdelacroix) [reply mattermost-plugin/server/plugin.go:56]: Oups, just saw that the comment had a line included, quickly fixing

Comment (mgdelacroix) [reply mattermost-plugin/server/plugin.go:56]: Done on 4d5d16a, now log happens right after the `New` call

**Pull request merged**